### PR TITLE
feat: Use dev tools chrome extension

### PR DESCRIPTION
**Pull request by Teapane**
Adds Chrome extension. 

Addresses #11

### CODE REVIEW

**Comment by Teapane** [comment]
@c0 This should be good to go! :smile:

File: package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "cz-conventional-changelog": "1.1.5",
     "enzyme": "2.0.0",
     "eslint": "^1.10.3",
+    "eslint-config-airbnb": "^5.0.1",
```
Comment (c0) [inline package.json]: Looks like this doubled up two lines down.

Comment (Teapane) [inline package.json]: :disappointed:  I'll amend, one second.

**Comment by c0** [comment]
:boom:

**Pull request merged**